float32 aileron
float32 elevator
float32 rudder
float32 throttle
int32 gear
float32 flaps
float32 speed_brakes